product_docs/docs/postgres_distributed_for_kubernetes/1/node_joins.mdx

```diff
@@ -7,35 +7,35 @@ When scaling up a PGD group by adding more data nodes or creating cross-region g
 it's crucial for new nodes to join the existing group properly. There are two methods for
 joining:
 
--   logical join:
+-   Logical join
 
       This method uses the [bdr.join_node_group](/pgd/latest/reference/nodes-management-interfaces#bdrjoin_node_group) function to integrate the new node into the PGD group.
-      It is important that the joining node should not contain any schemas or data present in the PGD group.
-      PGD recommends that the new database should only contain the BDR extension, as data synchronization occurs during
+      It's important that the joining node doesn't contain any schemas or data present in the PGD group.
+      We recommend that the new database contain only the BDR extension, as data synchronization occurs during
       the join.
 
--   physical join
+-   Physical join
 
-      This method uses the [bdr_init_physical](/pgd/latest/reference/nodes/#bdr_init_physical) command to speed up the joining process. We can prepare data in advance before executing `bdr_init_physical`.
+      This method uses the [bdr_init_physical](/pgd/latest/reference/nodes/#bdr_init_physical) command to speed up the joining process. You can prepare data in advance before executing `bdr_init_physical`.
 
-For more information about join method, refer to the documentation on [Creating and Joining PGD Groups](/pgd/latest/node_management/creating_and_joining/#creating-and-joining-pgd-groups).
+For more information about join methods, see [Creating and joining PGD groups](/pgd/latest/node_management/creating_and_joining/#creating-and-joining-pgd-groups).
 
 ## The initial group
 
-The PGD4K operator allows you to configure the join methods for nodes joining PGD groups,
+The PGD4K operator allows you to configure the join methods for nodes joining PGD groups
 as well as for groups joining other groups.
 
 When creating a single PGD group, you must configure it to create the *parent
 group* (`spec.pgd.parentGroup.create`=true). If you intend to create multiple PGD groups that
-will cross-join, at least one of these groups must be designated to form the parent group.
+will cross join, at least one of these groups must be designated to form the parent group.
 This group is referred to as the `initial` PGD group.
 
-The *first* node in the `initial` PGD group can either be restored from a backup
+The first node in the `initial` PGD group can either be restored from a backup
 (as described in the recovery documentation) or created from scratch. This node will always
-use a logical join to connect to the parent group it created. Conversely, the *first* node in
-the `non-initial` PGD group is used for cross-joining with other PGD groups.
+use a logical join to connect to the parent group it created. Conversely, the first node in
+the `non-initial` PGD group is used for cross joining with other PGD groups.
```

Comment:
It was this tagging that led me to question the tagging of initial. Is this part of code or is this a "first use" definition that would instead require italic?